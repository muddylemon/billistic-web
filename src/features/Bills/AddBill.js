import React from "react"
import "./bill_list.scss"
import { compose } from "recompose"

import { withExpenseHandlers, withExpenseData } from './Bill.hocs'

import Paper from '@material-ui/core/Paper'
// import Checkbox from '@material-ui/core/Checkbox'

export const AddBill = () => (
  <Paper>
  </Paper>
)

export default compose(
  withExpenseData,
  withExpenseHandlers
)(AddBill)
