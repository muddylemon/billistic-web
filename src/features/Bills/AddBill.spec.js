import React from "react"
import { AddBill } from "./AddBill"
import ShallowRenderer from "react-test-renderer/shallow"

describe('AddBill', () => {
  it('should render fine', () => {
    const tree = new ShallowRenderer().render(<AddBill />)
    expect(tree).toMatchSnapshot()
  })
})
