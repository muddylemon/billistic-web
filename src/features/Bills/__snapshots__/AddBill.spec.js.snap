// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddBill should render fine 1`] = `<WithStyles(Paper) />`;
